content/apidocs-mxsdk/apidocs/team-server-api.md

```diff
@@ -4,203 +4,411 @@ category: "API Documentation"
 menu_order: 65
 ---
 
+{{% alert type="warning" %}}
+The API description on this page refers to the new team server API implementation. You can access the documentation for the previous team server API implementation [here](old-version/team-server-api.md).
```

Comment:
Same as above – if possible, use version numbers.

Reply:
`here` should be changed to the name of the doc. (`For information about the implementation of team server API version xxx , see [Team Server API Version XXX](team-server-api-version-xxx).`)

Reply:
Removed warning as we will call this new API `App Repository API`